src/main/java/lotto/service/Converter.java

```diff
@@ -0,0 +1,45 @@
+package lotto.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import lotto.constant.GlobalConstant;
+import lotto.constant.WinningCondition;
+
+public class Converter {
+
+    public static int integerConvert(String number) {
```

Comment:
```suggestion
    public static int convertToInteger(String number) {
```
메서드명을 작성할 때는 동사형으로 시작해야한다는 코딩 컨벤션이 있어서 이렇게 적으면 좋을 것 같아요!

Reply:
헉 조언 감사합니다 동사형으로 메서드 사용해볼게요!